include/inviwo/core/properties/ordinalrefproperty.h

```diff
@@ -274,6 +281,45 @@ namespace util {
  */
 IVW_CORE_API OrdinalRefPropertyState<vec4> ordinalRefColor(
     InvalidationLevel invalidationLevel = InvalidationLevel::InvalidOutput);
+
+/**
+ * A factory function for configuring a OrdinalRefProperty representing a generic vector, with a
+ * symmetric range around zero, and Ignored boundary constraints. The invalidation level defaults to
+ * InvalidOutput, and the property semantics to SpinBox.
+ * @param value the default value for the property
+ * @param minMax used to construct the range of the property like min = T{-minMax}, max = T{minMax}.
+ * The constraint behavior will be Ignore.
+ */
+template <typename T = double, typename U = T>
+OrdinalRefPropertyState<T> ordinalRefSymmetricVector(const U& minMax = U{100}) {
+    using V = util::value_type_t<T>;
+    if constexpr (std::is_floating_point_v<util::value_type_t<T>>) {
+        return {T{-minMax},
+                ConstraintBehavior::Ignore,
+                T{minMax},
+                ConstraintBehavior::Ignore,
+                T{static_cast<V>(0.1)},
+                InvalidationLevel::InvalidOutput,
+                PropertySemantics::SpinBox};
+    } else if constexpr (std::is_signed_v<util::value_type_t<T>>) {
+        return {T{-minMax},
+                ConstraintBehavior::Ignore,
+                T{minMax},
+                ConstraintBehavior::Ignore,
+                T{static_cast<V>(1)},
```

Comment:
Is it intentional to have 1 here and 0.1 above?
```suggestion
                T{static_cast<V>(0.01 * minMax)},
```